bridge/slack/handlers.go

```diff
@@ -132,7 +132,18 @@ func (b *Bslack) skipMessageEvent(ev *slack.MessageEvent) bool {
 		ev.SubMessage.Edited == nil {
 		return true
 	}
-	return false
+
+	return b.filesCached(ev.Files)
+}
+
+func (b *Bslack) filesCached(files []slack.File) bool {
+	for _, f := range files {
+		f := f
```

Comment:
I'd rather that this be:
```go
for i := range files {
	if !b.fileCached(&files[i]) {
		return false
	}
}
```
Copying is not useful here and it's also introducing a shadow variable.

Reply:
It's based on the suggestions on https://github.com/kyoh86/scopelint
But don't mind one way or the other.